cltk/corpus/arabic/alphabet.py

```diff
@@ -0,0 +1,16 @@
+"""Arabic alphabet"""
+
+
+#Arabic digits from 0 to 9
+DIGITS = ['۰', '۱', '۲', '۳', '٤', '۵', '٦', '۷', '۸', '۹']
+
+#Diacritics
+SHORT_VOWELS = ['َ', 'ِ', 'ُ']
+
+#The Persian Alphabet
+LONG_VOWELS = ['ا', 'و', 'ي']
+
+CONSONANTS = ['ب', 'ت', 'ث', 'ج', 'ح', 'خ', 'د', 'ذ', 'ر', 'ز', 'س', 'ش', 'ص', 'ض', 'ط', 'ظ', 'ع', 'غ', 'ف', 'ق', 'ک', 'ل', 'م', 'ن', 'ه']
```

Comment:
I know the distinction is not made by those who speak Arabic. But the alphabet can still be classified based on the model set by other languages, just for the sake of uniformity. 
https://en.wikipedia.org/wiki/Arabic_alphabet#Vowels

I was not sure if hamza is used in Arabic text. I had added it for urdu/alphabet.py, but deliberately left it out here. Should it be added?